presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.pinot;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class TestPinotUtils
+{
+    @Test
+    public void testParseTimestamp()
+    {
+        long epoch = PinotUtils.parseTimestamp("1.672528152E12");
```

Comment:
Please import `import static com.facebook.presto.pinot.PinotUtils.parseTimestamp;` so that we can do

`long epoch = parseTimestamp("1.672528152E12");`

here and below